### ISSUE TITLE
modifiable max response length?

### ISSUE THREAD

**Issue by BBC-Esq** [LostRuins/lite.koboldai.net#83]
I noticed when playing with Kobold that the max response length slider bar is maxed at 512 and there isn't a setting within the quicklaunch GUI that allows me to change this.  Apparently, once I start kobold it can be changed within "settings" by manually typing in a different number - e.g. 1024 - but it wasn't immediately apparent to me because I assumed the slider bar represented anything/everything that the program would accept.  It'd be nice to be able to change it within the launcher GUI from the beginning and/or have a higher amount than 512 in the "settings" even after Kobold starts up.

**Comment by LostRuins** [comment]
As you said, you can overwrite the number in the input box if you need values greater than 512.

If you need to adjust the slider, you can use this usermod
```
document.getElementById('max_length_slide').max = 9999;
```
![image](https://github.com/user-attachments/assets/a12b19af-d3e7-4453-8cd4-3d590f3ed147)

**Comment by BBC-Esq** [comment]
Thanks I'll check that out in the meantime.  Is this not something that would be a beneficial change?

**Comment by LostRuins** [comment]
Not really. It's misleading to new users, because an increased gen amount actually reduces the available context since space needs to be reserved for the output. That's why the slider max by default is 512 - people can always increase it further if they need to.
